[2023-09-12T17:35:48.356+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.say_hello scheduled__2023-08-31T00:00:00+00:00 [queued]>
[2023-09-12T17:35:48.407+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.say_hello scheduled__2023-08-31T00:00:00+00:00 [queued]>
[2023-09-12T17:35:48.412+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-09-12T17:35:48.487+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): say_hello> on 2023-08-31 00:00:00+00:00
[2023-09-12T17:35:48.506+0000] {standard_task_runner.py:57} INFO - Started process 2149 to run task
[2023-09-12T17:35:48.551+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'say_hello', 'scheduled__2023-08-31T00:00:00+00:00', '--job-id', '1004', '--raw', '--subdir', 'DAGS_FOLDER/1_my_first_dag.py', '--cfg-path', '/tmp/tmpisx9ltpg']
[2023-09-12T17:35:48.567+0000] {standard_task_runner.py:85} INFO - Job 1004: Subtask say_hello
[2023-09-12T17:35:48.748+0000] {task_command.py:415} INFO - Running <TaskInstance: my_first_dag.say_hello scheduled__2023-08-31T00:00:00+00:00 [running]> on host 0ab7970e0956
[2023-09-12T17:35:48.998+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='say_hello' AIRFLOW_CTX_EXECUTION_DATE='2023-08-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-31T00:00:00+00:00'
[2023-09-12T17:35:48.999+0000] {logging_mixin.py:151} INFO - hello world
[2023-09-12T17:35:48.999+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-09-12T17:35:49.017+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=say_hello, execution_date=20230831T000000, start_date=20230912T173548, end_date=20230912T173549
[2023-09-12T17:35:49.104+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-12T17:35:49.130+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
